spec/advance_enumerable_spec.rb

```diff
@@ -0,0 +1,163 @@
+require '../advace_enumerable.rb'
```

Comment:
Using `require_relative` for files in your directory instead of `require` is a ruby convention.

"Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."

I had to change it in your repo to be able to run the tests.